src/Control/Middleware/CanonicalURLMiddleware.php

```diff
@@ -81,6 +81,8 @@ class CanonicalURLMiddleware implements HTTPMiddleware
      * @var array|bool
      */
     protected $enabledEnvs = [
+        CoreKernel::DEV,
+        CoreKernel::TEST,
```

Comment:
We can't make this change outside of a major release without breaking some peoples' workflows.
Depending on what the issue actually is that you're trying to resolve, there might be another way to tackle it - but if this *does* turn out to be the correct way to resolve this problem, the PR will need to target the `6` branch and have an associated PR for the CMS 6 changelog.

Reply:
Updated now!

Reply:
Thanks! Can you please open a pull request to the `6` branch of developer docs repo to update the CMS 6 changelog (you may need to add a new file for it)?
Please also create an issue and link the two PRs to it so I can track this.